compute/instance.tf

```diff
@@ -0,0 +1,186 @@
+
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-*"]
+  }
+
+  /*data "aws_vpc" "main" {
+    name = "DevOps_VPC"
+  }*/
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+/*data "aws_subnet" "private" {
+  cidr_block = var.private_cidr[count.index]
+}*/
```

Comment:
Please don't leave commented-out blocks of code. This is considered as a bad style